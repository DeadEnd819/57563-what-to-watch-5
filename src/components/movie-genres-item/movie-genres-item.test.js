import React from "react";
import renderer from "react-test-renderer";
import MovieGenresItem from "./movie-genres-item";

const noop = ()=>{};

describe(`Render PlayerButton`, () => {
  it(`Should MovieGenresItem activeGenre render correctly`, () => {
    const tree = renderer
      .create(
          <MovieGenresItem genre={`Comedy`} activeGenre={`Comedy`} onGenreClick={noop}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should MovieGenresItem render correctly`, () => {
    const tree = renderer
      .create(
          <MovieGenresItem genre={`Fantasy`} activeGenre={`Crime`} onGenreClick={noop}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
