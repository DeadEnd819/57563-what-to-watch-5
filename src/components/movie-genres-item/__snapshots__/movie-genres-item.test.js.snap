// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PlayerButton Should MovieGenresItem activeGenre render correctly 1`] = `
<li
  className="catalog__genres-item catalog__genres-item--active"
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    Comedy
  </a>
</li>
`;

exports[`Render PlayerButton Should MovieGenresItem render correctly 1`] = `
<li
  className="catalog__genres-item "
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    Fantasy
  </a>
</li>
`;
